apps/core/components/rangeDatePicker/RangeDatePicker.js

```diff
@@ -0,0 +1,117 @@
+/* @flow */
```

Comment:
My OCD ticks from this flow annotation 😱

Reply:
Fixed